README.rst

```diff
@@ -136,6 +138,26 @@ text dealing with spelling errors, country name variations, etc.:
     E.g., ``Puerto Rico`` is mapped to ``United States`` and ``Greenland`` to ``Denmark``.
 
 
+Resource loading
+~~~~~~~~~~~~~~~~
+
+Resources for cities and regions aren't all loaded when you import ``TextScrubber``, they're loaded on the fly. This
+means that the first time you do a query it can take a while. The second time around it will be much faster. You can
```

Comment:
this is per `country` right? that first time a country is encountered it will be slow and subsequent queries with that country will be faster ? if yes, maybe you can rephrase it.

Reply:
Yes, it's per country. Will update the text